src/graphql/Bug.graphql

```diff
@@ -271,6 +271,27 @@ input BugSearch {
   whiteboards: [String]
 }
 
+type Comment {
+  # The globally unique ID for the comment.
+  id: ID!
+  # The ID of the bug that this comment is on.
+  bugId: ID!
```

Comment:
`bugId`s are `Int`s not `ID`s. I'm not sure if the same applies to `id`s for comments.

Reply:
You're right. Also, by navigating to https://bugzilla.mozilla.org/rest/bug/1010/comment for example, you can see the `id` field returned as an `Int`.